app.json

```diff
@@ -61,7 +61,7 @@
       "url": "heroku/nodejs"
     },
     {
-      "url": "heroku/python"
+      "url": "https://github.com/kennethreitz/heroku-buildpack-pipenv"
```

Comment:
This repo does not exist. Also, heroku works with pipenv out of the box, shouldn't need a special buildpack.